Dangerfile

```diff
@@ -0,0 +1,150 @@
+# frozen_string_literal: true
+
+# A Danger script which checks out dependent projects and builds them against the
+# current (installed) version of this project.
+#
+# XXX: This script can be improved further. Open points:
+# - The script assumes that the parent project resides in the current directory
+#   and has already been installed into the local Maven repository.
+# - The script assumes this project and all downstream dependencies are Maven
+#   projects. Think of ways to generalize this.
```

Comment:
Is this likely seeing as this project is essentially a Maven project? Or do you mean as a way to apply this script to other possible parent projects?